tests/functional/provisioning/test_dev_path_mapping_file.py

```diff
@@ -792,3 +793,93 @@ def test_dev_path_mapping_heketi_node_delete(self):
             use_percent, use_percent_after,
             "Failed to execute IO's in the app pod {} after respin".format(
                 pod_name))
+
+    @pytest.mark.tier2
+    @podcmd.GlustoPod()
+    def test_dev_path_volume_expansion(self):
+        """Validate dev path mapping for file volumes expansion"""
+
+        pvc_size, pvc_amount, expand_pvc_size = 1, 2, 2
+        vol_names, pvc_names = [], []
+
+        # Create PVC's
+        sc_name = self.create_storage_class(allow_volume_expansion=True)
+
+        # openshift_ops.wait_for_pvcs_be_bound(self.node, pvc_names)
+        pvc_names = self.create_and_wait_for_pvcs(
+            pvc_size=pvc_size, pvc_amount= pvc_amount, sc_name=sc_name)
+        dc_name = self.create_dcs_with_pvc(pvc_names)
+        self.validate_file_volumes_count(
+            self.h_node, self.h_server, self.node_ip)
+
+        for pvc_name in pvc_names:
+            pv_name = openshift_ops.get_pv_name_from_pvc(self.node, pvc_name)
+            volume_name = openshift_ops.get_vol_names_from_pv(
+                self.node, pv_name)
+            vol_names.append(volume_name)
+
+        # Get pvs info
+        pvs_info_before = openshift_storage_libs.get_pvs_info(
+            self.node, self.node_ip, self.devices_list, raise_on_error=False)
```

Comment:
why `raise_on_error=False` ?

Reply:
used it for testing. done

Reply:
@srivickynesh, so the `raise_on_error=False` is not required and can be removed?